app/mailers/account_transaction_report_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class AccountTransactionReportMailer < BaseMailer
```

Comment:
Instead of creating a separate mailer, can we just use the existing `CsvReportMailer` and pass the report in?